### PR TITLE
setuptools-markdown 0.4.1 :snowflake: 

### PR DESCRIPTION
**Pull request by AndrewOsip**
setuptools-markdown 0.4.1 :snowflake:

**Destination channel:** Snowflake

- [PKG-5938](https://anaconda.atlassian.net/browse/PKG-5938) 
- [Upstream repository](https://github.com/msabramo/setuptools-markdown)

[PKG-5938]: https://anaconda.atlassian.net/browse/PKG-5938?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tombenes — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  noarch: python
 
 requirements:
-  build:
-    - pip
+  host:
     - python
-    - setuptools
```
Comment (lorepirri) [inline recipe/meta.yaml:24]: `setuptools` still needed

**Comment by lorepirri** [comment]
Upstream: https://github.com/msabramo/setuptools-markdown/commit/34ed6276e0c8bcefcd967bd8db138546a270bdf4

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
   license_file: LICENSE-MIT
-  summary: Use Markdown for your project description
+  description: Markdown for project description
   dev_url: https://github.com/msabramo/setuptools-markdown
+  doc_url: https://github.com/msabramo/setuptools-markdown/blob/master/README.rst
 
 extra:
   recipe-maintainers:
-    - nicoddemus
+    - snegireff
```
Comment (lorepirri) [inline recipe/meta.yaml]: usually we do not update the maintainer unless it's a new feedstock or a complete rewrite

**Review by lorepirri — APPROVED** [review]
LGTM, Thanks for improving it

**Pull request merged**